waltid-libraries/waltid-openid4vc/src/commonMain/kotlin/id/walt/oid4vc/providers/OpenIDCredentialIssuer.kt

```diff
@@ -34,7 +44,7 @@ abstract class OpenIDCredentialIssuer(
         )
     private var _supportedCredentialFormats: Set<CredentialFormat>? = null
     val supportedCredentialFormats
```

Comment:
ide suggests `supportedCredentialFormats` could be private, is it the case?

Reply:
may be useful in deriving class